```
2017-11-09 11:35:33,441 23823 [1;32m[1;49mINFO[0m openupgrade hr_attendance: test_hr_attendance_view_removed (test_view_removal.TestHrAttendanceViewRemoved)
2017-11-09 11:35:33,446 23823 [1;31m[1;49mERROR[0m openupgrade hr_attendance: FAIL
2017-11-09 11:35:33,450 23823 [1;32m[1;49mINFO[0m openupgrade hr_attendance: ======================================================================
2017-11-09 11:35:33,451 23823 [1;31m[1;49mERROR[0m openupgrade hr_attendance: FAIL: test_hr_attendance_view_removed (test_view_removal.TestHrAttendanceViewRemoved)
2017-11-09 11:35:33,451 23823 [1;31m[1;49mERROR[0m openupgrade hr_attendance: Traceback (most recent call last):
2017-11-09 11:35:33,451 23823 [1;31m[1;49mERROR[0m openupgrade hr_attendance: `   File "/home/travis/build/OCA/OpenUpgrade/addons/hr_attendance/migrations/9.0.1.1/tests/test_view_removal.py", line 8, in test_hr_attendance_view_removed
2017-11-09 11:35:33,451 23823 [1;31m[1;49mERROR[0m openupgrade hr_attendance: `     self.assertFalse(self.env.ref('hr_attendance.hr_attendace_group'))
2017-11-09 11:35:33,451 23823 [1;31m[1;49mERROR[0m openupgrade hr_attendance: ` AssertionError: ir.ui.view(536,) is not false
2017-11-09 11:35:33,451 23823 [1;32m[1;49mINFO[0m openupgrade hr_attendance: Ran 1 test in 0.009s
```

--- a/addons/hr_attendance/migrations/9.0.1.1/openupgrade_analysis.txt
+++ b/addons/hr_attendance/migrations/9.0.1.1/openupgrade_analysis.txt
@@ -11,7 +11,11 @@ DEL ir.ui.menu: hr_attendance.menu_hr_attendance_graph
 DEL ir.ui.menu: hr_attendance.menu_open_view_attendance_reason
 NEW ir.ui.view: hr_attendance.hr_department_view_kanban
 NEW ir.ui.view: hr_attendance.view_hr_attendance_pivot
+
 DEL ir.ui.view: hr_attendance.hr_attendace_group
+# Done, set view to noupdate = false so that it is removed and does not block
+# the removal of the views that it inherits from
+
 DEL ir.ui.view: hr_attendance.hr_attendance_employee
 NEW res.groups: base.group_hr_manager
 DEL res.groups: base.group_hr_attendance

--- a/addons/hr_attendance/migrations/9.0.1.1/pre-migration.py
+++ b/addons/hr_attendance/migrations/9.0.1.1/pre-migration.py
@@ -1,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Opener B.V. <https://opener.amsterdam>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    env.cr.execute(
+        """ UPDATE ir_model_data SET noupdate = false
+        WHERE module = 'hr_attendance' AND name = 'hr_attendace_group' """)

--- a/openerp/openupgrade/tests_deferred/tests_deferred.py
+++ b/openerp/openupgrade/tests_deferred/tests_deferred.py
@@ -14,3 +14,9 @@ class TestDeferred(common.TransactionCase):
                 ('res_field', '=', 'image'),
                 ('res_name', '=', 'Axelor...'),
             ]).datas)
+
+    def test_hr_attendance_view_removed(self):
+        """ This view, which had the noupdate flag, is now successfully removed
+        """
+        with self.assertRaisesRegexp(ValueError, 'External ID not found'):
+            self.env.ref('hr_attendance.hr_attendace_group')

